src/main/java/aquality/selenium/core/application/AqualityModule.java

```diff
@@ -3,8 +3,10 @@
 import aquality.selenium.core.logging.Logger;
```

Comment:
should not we add the settings file in the src/main/resources with this PR?

Reply:
we added customizations of this file, and there are tests which check, that if settings.json file is absent and profile is empty, configuration will be got from default file.